src/store/getters/state-getters.ts

```diff
@@ -848,14 +871,14 @@ export const getFeePrices = (state: StateIF): FeesIF => {
   return state.stateModel.feePrices
 }
 
-/** True if the minimum share classes requirements are not met. */
-export const invalidMinimumShareClass = (state: StateIF): boolean => {
+/** True when the minimum share classes requirements are met. */
+export const hasMinimumShareClass = (state: StateIF): boolean => {
   const shareClasses = state.stateModel.shareStructureStep.shareClasses
 
   // Filter out REMOVED class actions
   const currentShareClasses = shareClasses.filter(x => x.action !== ActionTypes.REMOVED)
 
-  return (currentShareClasses.length < 1)
+  return (currentShareClasses.length > 0)
```

Comment:
I changed this to make it consistent with other, similar getters, and because double-negatives are hard to understand.